src/modules/network.cpp

```diff
@@ -158,6 +158,7 @@ namespace modules {
       label->replace_token("%local_ip6%", network->ip6());
       label->replace_token("%upspeed%", upspeed);
       label->replace_token("%downspeed%", downspeed);
+      label->replace_token("%netspeed%", upspeed + downspeed);
```

Comment:
This doesn't work. `uspeed` and `downspeed` are strings, this will just show nonsense. For example if I have 1KB download and 5KB upload, this will show `1KB5KB`.

Reply:
Oh,So we have to create another function for netspeed everywhere?

Reply:
@patrick96 Can you check now.